### DISCUSSION TITLE
I open-sourced aws-parquet an object-oriented interface to awswrangler

### DISCUSSION THREAD

**Discussion by marwan116** [aws/aws-sdk-pandas#2313] [Show and tell]
Check out [aws-parquet](https://github.com/marwan116/aws-parquet) as an object-oriented approach to working with awswrangler and parquet data. It handles AWS S3 reads/writes, AWS Glue catalog updates and AWS Athena queries by providing a simple and intuitive interface

Looking forward to your feedback

**Comment by jaidisido** [comment]
Just saw this, kudos @marwan116! Love the encapsulation behind the dataset construct and its interface

**Reply by marwan116** [reply]
Thank you!
